src/main/java/com/google/swarm/tokenization/beam/ConvertCSVRecordToDLPRow.java

```diff
@@ -50,37 +50,41 @@ public ConvertCSVRecordToDLPRow(
 
   @ProcessElement
   public void processElement(ProcessContext context) throws IOException {
-    Table.Row.Builder rowBuilder = Table.Row.newBuilder();
-    String input = Objects.requireNonNull(context.element().getValue());
-    String fileName = context.element().getKey();
-    Map<String, List<String>> headers = context.sideInput(header);
-    List<String> csvHeader = headers.get(fileName);
-    if (csvHeader == null) {
-      throw new RuntimeException(
-          "Unable to find header row for fileName: "
-              + fileName
-              + ". The side input only contains header for "
-              + headers.keySet());
-    }
-
-    if (columnDelimiter != null) {
-
-      List<String> values = Util.parseLine(input, columnDelimiter, '"');
-      if (values.size() == csvHeader.size()) {
-        values.forEach(
-            value -> rowBuilder.addValues(Value.newBuilder().setStringValue(value).build()));
-        context.output(KV.of(fileName, rowBuilder.build()));
-
-      } else {
-        LOG.warn(
-            "Rows in {} must have the same number of items {} as there are headers {}",
-            fileName,
-            values.size(),
-            csvHeader.size());
-      }
-    } else {
-      rowBuilder.addValues(Value.newBuilder().setStringValue(input).build());
-      context.output(KV.of(fileName, rowBuilder.build()));
+    try {
+        Table.Row.Builder rowBuilder = Table.Row.newBuilder();
+        String input = Objects.requireNonNull(context.element().getValue());
+        String fileName = context.element().getKey();
+        Map<String, List<String>> headers = context.sideInput(header);
+        List<String> csvHeader = headers.get(fileName);
+        if (csvHeader == null) {
+          throw new RuntimeException(
+              "Unable to find header row for fileName: "
+                  + fileName
+                  + ". The side input only contains header for "
+                  + headers.keySet());
+        }
+    
+        if (columnDelimiter != null) {
+    
+          List<String> values = Util.parseLine(input, columnDelimiter, '"');
+          if (values.size() == csvHeader.size()) {
+            values.forEach(
+                value -> rowBuilder.addValues(Value.newBuilder().setStringValue(value).build()));
+            context.output(KV.of(fileName, rowBuilder.build()));
+    
+          } else {
+            LOG.warn(
+                "Rows in {} must have the same number of items {} as there are headers {}",
+                fileName,
+                values.size(),
+                csvHeader.size());
+          }
+        } else {
+          rowBuilder.addValues(Value.newBuilder().setStringValue(input).build());
+          context.output(KV.of(fileName, rowBuilder.build()));
+        }
+    } catch (IllegalArgumentException e) {
+        LOG.info(e.getMessage());
     }
```

Comment:
When will this happen? What is the impact of this log and swallow on the pipeline?

Reply:
Fixed